README.md

```diff
@@ -1,3 +1,6 @@
 # sandbox
 
-Updateing Sandbox
```

Comment:
It should be more informative

Reply:
For now, It is ok, As it is just setup
I will add more info. in upcoming PRs